pocs/focuser/simulator.py

```diff
@@ -15,9 +15,32 @@ def __init__(self,
                  *args, **kwargs):
         super().__init__(*args, name=name, port=port, **kwargs)
         self.logger.debug("Initialising simulator focuser")
+        self._is_moving = False
         self.connect()
         self.logger.info("{} initialised".format(self))
 
+##################################################################################################
+# Properties
+##################################################################################################
+
+    @property
+    def min_position(self):
+        """
+        Returns position of close limit of focus travel, in encoder units
+        """
+        return self._min_position
+
+    @property
+    def max_position(self):
+        """
+        Returns position of far limit of focus travel, in encoder units
+        """
+        return self._max_position
+
+    @property
+    def is_moving(self):
```

Comment:
This class inherits from `AbstractFocuser` so this property isn't needed here?

Reply:
`AbstractFocuser` has `is_moving` as an `abstractmethod`. Arguably should just implement it there, though.